get_aqi.py

```diff
@@ -1,32 +1,55 @@
 #!/usr/bin/python3
 
+import os
 import argparse
 import datetime
 import time
+import logging
 from collections import deque
+from threading import Thread
 
 import paho.mqtt.publish
+from prometheus_client import start_http_server, Gauge, Histogram
+
 from aqipy import aqi_cn, aqi_us, caqi_eu
 from sds011 import SDS011
 
+logging.basicConfig(
+    format='%(asctime)s.%(msecs)03d %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    handlers=[logging.FileHandler("sds011_exporter.log"),
+              logging.StreamHandler()],
+    datefmt='%Y-%m-%d %H:%M:%S')
+
+DEBUG = os.getenv('DEBUG', 'false') == 'true'
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Measure air quality using an SDS011 sensor.")
 
-    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="EU", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
-    parser.add_argument("--delay", "-d", default=1200, metavar="SECONDS", type=int, help="seconds to pause after getting data with the sensor before taking another measure (default: 1200, ie. 20 minutes)")
+    parser.add_argument("-b", "--bind", metavar='ADDRESS', default='0.0.0.0', help="Specify alternate bind address [default: 0.0.0.0]")
+    parser.add_argument("-p", "--port", metavar='PORT', default=8000, type=int, help="Specify alternate port [default: 8000]")
+    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="US", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
```

Comment:
As I use the CAQI index to compute the AQI level, I prefer to keep the default to EU.
You can tweak this behavior with the `-c` or `--country` option when running the script

```suggestion
    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="EU", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
```